<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transformer_dlg</class>
 <widget class="QDialog" name="transformer_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformer Properties</string>
  </property>
  <widget class="QLabel" name="lbl_current_output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Output</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transformer Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_transformer_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_power_max">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_voltage_iso">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage - Isolation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_current_output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_voltage_secondary">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_voltage_secondary">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage - Secondary</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_inductance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inductance</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_power_max">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Power - Max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_inductance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_turns_ratio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Turns Ratio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_voltage_isolation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_voltage_primary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage - Primary</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_voltage_primary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_turns_ratio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_center_tap">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Center Tap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>box_transformer_type</tabstop>
  <tabstop>cbx_center_tap</tabstop>
  <tabstop>box_voltage_primary</tabstop>
  <tabstop>box_voltage_secondary</tabstop>
  <tabstop>box_current_output</tabstop>
  <tabstop>box_power_max</tabstop>
  <tabstop>box_voltage_isolation</tabstop>
  <tabstop>box_inductance</tabstop>
  <tabstop>box_turns_ratio</tabstop>
  <tabstop>btn_OK</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
